src/elements/content-sidebar/__tests__/ActivitySidebar.test.js

```diff
@@ -1578,19 +1581,68 @@ describe('elements/content-sidebar/ActivitySidebar', () => {
                 const instance = wrapper.instance();
                 expect(instance.renderActivitySidebarFilter()).toBe(null);
             });
+
+            test('when activityFeed.newThreadedReplies is disabled and no filter options list is provided', () => {
+                const wrapper = getWrapper({
+                    features: {
+                        activityFeed: {
+                            newThreadedReplies: { enabled: false },
+                        },
+                    },
+                });
+                const instance = wrapper.instance();
+                expect(instance.renderActivitySidebarFilter()).toBe(null);
+            });
+
+            test('when activityFeed.newThreadedReplies is disabled and filter options list is provided', () => {
+                const wrapper = getWrapper({
+                    activityFilterOptions: ['all', 'open', 'resolved', 'tasks'],
+                });
+                const instance = wrapper.instance();
+                expect(instance.renderActivitySidebarFilter()).toBe(null);
+            });
+
+            test('when activityFeed.newThreadedReplies is enabled and no filter options list is provided', () => {
```

Comment:
Do we also need to test for `when activityFeed.newThreadedReplies is enabled and filter options list is provided`? Also, would it be easier to group these tests together in a `test.each...` since they're similar?